setup.py

```diff
@@ -37,8 +37,11 @@ def _dbt_duckdb_version():
     packages=find_namespace_packages(include=["dbt", "dbt.*"]),
     include_package_data=True,
     install_requires=[
-        "dbt-core~=1.7.0",
+        "dbt-common>=1,<2",
+        "dbt-adapters>=1,<2",
         "duckdb>=0.7.0",
+        # add dbt-core to ensure backwards compatibility of installation, this is not a functional dependency
+        "dbt-core>=1.8.0b1",
```

Comment:
Even though `dbt-core` is no longer a functional dependency of adapters, we're going to keep including `dbt-core` in `setup.py` to prevent surprising/breaking changes to the behavior of `pip install dbt-duckdb` on the day of the v1.8 final release.

`dbt-duck` shouldn't import anything from `dbt-core` directly, only from the interfaces defined in `dbt-core` + `dbt-adapters`.